code/game/turfs/simulated/floor.dm

```diff
@@ -43,12 +42,17 @@ var/list/wood_icons = list("wood","wood-broken")
 		icon_regular_floor = "floor"
 	else
 		icon_regular_floor = icon_state
+	if(floor_tile)
+		builtin_tile = new floor_tile
+
+	spawn(5)
+		update_visuals()

```

Comment:
Hmm; this isn't on TG; what's the reason for it here, in addition to the delay on it?